utils.js

```diff
@@ -39,8 +58,8 @@ export async function runInRepo({ repo, workspace, folder, build, test, override
   test = pnpmCommand(test)
 
   if( !folder ) {
-    // Use the repository name as the folder
-    folder = repo.split('/')[1].slice(0,-4)
+    // Use the repository name as the folder, omit possible org name
+    folder = repo.substring(repo.lastIndexOf('/')+1)
```

Comment:
this isn't ideal if you pass in a custom repo with .git at the end, but feels like a safe bet everyone will be using github for this.